lib/rom/rails/configuration.rb

```diff
@@ -1,33 +1,36 @@
 module ROM
   module Rails
     class Configuration
-      attr_reader :config, :setup, :env
+      attr_reader :repos
```

Comment:
It'd be nice to use `repositories` here. I know it's longer but that's how it's called in ROM. I'm probably guilty of using the word "repos" when chatting about it, please excuse me :)